config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index], controller: 'users/posts'
+  end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :posts, only: [:show]
```

Comment:
- Good job done so far, but this route is not needed as you already have the route above, please remove this 🙏